visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.ts

```diff
@@ -77,26 +77,75 @@ export class ArtificialIntelligenceController implements FilesSelectionSubscribe
 			return
 		}
 
-		// Thresholds are derived for Java maps only (currently)
-		if (!this.hasJavaFiles(fileState.file.map)) {
-			return
+		this.mainProgrammingLanguage = this.getMostFrequentLanguage(fileState.file.map)
```

Comment:
The Risk Profile and the inspection for suspicious metrics happens for the most frequent programming language in the project. We should discuss to let the user choose the programming language for the calculations.

Reply:
is it indicated for the user that it is using the most frequent language?

Reply:
No, good point. Thanks.